doc/manual/manual.tex

```diff
@@ -5057,6 +5057,52 @@ \subsubsection{Artificial viscosity smoothing}
 
 
 
+\subsubsection{Tracking finite strain}
+\label{sec:finite-strain}
+\textit{This section was contributed by Juliane Dannberg}
+
+In many geophysical settings, material properties, and in particular the rheology, do not only depend
+on the current temperature, pressure and strain rate, but also on the history of the system. 
+This can be incorporated in a geodynamic model by tracking the strain that accumulates over time
+at every point in the model. Tracking this finite strain can either be done by using compositional
```

Comment:
There are other factors that describe history, so maybe the following would be a better sentence:

"This can be incorporated in \aspect{} models by tracking history variables through compositional fields. In this cookbook, we will show how to do this by tracking the strain that accumulates over time at every (Lagrangian) point in the model."